docs/guides/technical-docs-seo-guide.rst

```diff
@@ -0,0 +1,278 @@
+to help make sites easier to understand for search engines and rank better.
+To summarize some of the most important aspects
+as they apply to technical documentation, your site should:
+
+* Use descriptive and accurate titles in the HTML ``<title>`` tag.
+* Ensure your URLs are descriptive. They are displayed in search results.
+* Avoid low content pages or pages with very little original content.
+* Avoid tactics that attempt to increase your search engine ranking
+  without actually improving content.
+* Google specifically `warns about automatically generated content`_
+  although this applies primarily to keyword stuffing and low value content.
+  High quality documentation generated from source code
+  (eg. auto generated API documentation) seems OK.
+
+  .. _warns about automatically generated content: https://support.google.com/webmasters/answer/2721306
+
+While both Google and Bing discuss site performance as an important factor
+in search result ranking, this guide is not going to discuss it in detail.
+Most technical documentation that uses Sphinx or Read the Docs
+generates static HTML and the performance is typically decent
+relative to most of the internet.
+
+
+Optimizing your docs for search engine spiders
+----------------------------------------------
+
+Once a crawler or spider finds your site, it will follow links and redirects
+in an attempt to find any and all pages on your site.
+While there are a few ways to guide the search engine in its crawl
+for example by using a sitemap or a robots.txt file which we'll discuss shortly,
+the most important thing is making sure the spider can follow links on your site
+and get to all your pages.
+
+Avoid orphan pages
+~~~~~~~~~~~~~~~~~~
+
+Sphinx calls pages that don't have links to them "orphans"
+and will throw a warning while building documentation that contains an orphan
+unless the warning is silenced with the :ref:`orphan directive <sphinx:metadata>`:
+
+::
+
+    $ make html
+    sphinx-build -b html -d _build/doctrees . _build/html
+    Running Sphinx v1.8.5
+    ...
+    checking consistency... /path/to/file.rst: WARNING: document isn't included in any toctree
+    done
+    ...
+    build finished with problems, 1 warning.
+
+You can make all Sphinx warnings into errors during your build process
+by setting ``SPHINXOPTS = -W --keep-going`` in your Sphinx Makefile.
+
+
+Avoid uncrawlable content
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+While typically this isn't a problem with technical documentation,
+try to avoid content that is "hidden" from search engines.
+This includes content hidden in images or videos
+which the crawler may not understand.
+For example, if you do have a video in your docs,
+make sure the rest of that page describes the content of the video.
+
+When using images, make sure to set the image alt text or set a caption on figures.
+For Sphinx, the image and figure directives support this:
+
+.. sourcecode:: rst
+
+    .. image:: your-image.png
+        :align: right
```

Comment:
I think we can remove the `align` option from here so it's cleaner, since that option is not related with what we are trying to show in the example.